src/AccordionItem/__snapshots__/accordion-item.spec.js.snap

```diff
@@ -54,25 +54,28 @@ exports[`AccordionItem renders correctly with accordion true 1`] = `
 </div>
 `;
 
-exports[`AccordionItem renders with different className 1`] = `
+exports[`AccordionItem renders correctly with other blocks inside 1`] = `
 <div
-  className="testCSSClass"
+  className="accordion__item"
 >
   <div
     ariaControls="accordion__body-mock-uuid-hash"
     expanded={false}
     id="accordion__title-mock-uuid-hash"
     onClick={[Function]}
-    role="tab"
+    role="button"
```

Comment:
Hmmm that's a good point. That must be a very common issue, I wonder how other people are dealing with this.